docs/src/appendix/library.md

```diff
@@ -264,14 +265,15 @@ Modules = [Oceananigans, Oceananigans.AbstractOperations]
 Private = false
 Pages   = [
     "AbstractOperations/AbstractOperations.jl",
-    "AbstractOperations/unary_operations.jl",
+    "AbstractOperations/at.jl",
     "AbstractOperations/binary_operations.jl",
-    "AbstractOperations/multiary_operations.jl",
+    "AbstractOperations/broadcasting_abstract_operations.jl",
```

Comment:
I don't think this file contains any user-facing functions --- does it?

Reply:
You're right, it does not. I thought we were listing all files that didn't contain experimental features regardless of them having user-facing function or not. I'll remove that one.

Reply:
I think it might make sense to only include docstrings in the documentation for functions that users may find useful. There could be an advantage to limiting the number of functions that show up; that way users can more easily browse through them. Including docstrings for functions that only have internal use may not be helpful. What do you think?

Reply:
I agree!